### PR TITLE
Upgrade to 0.29

### PR DESCRIPTION
**Pull request by lizconlan**
Connects to #38

### CODE REVIEW

File: lib/views/request/_bubble.html.erb [hunk 1]
```diff
@@ -5,6 +5,7 @@
         <h3>
           <%= n_('{{count}} Attachment',
                  '{{count}} Attachments',
+                 '{{count}} Attachments',
```
Comment (garethrees) [inline lib/views/request/_bubble.html.erb]: +1 in principle, but a few questions:

* Do we actually support this?
* Have we done it elsewhere?
* Will this prevent the use of core translations? (i.e. they'd _have_ to provide a theme translation for this to work?)

Comment (lizconlan) [reply lib/views/request/_bubble.html.erb]: Transifex supports this - all the po files have a Plural Forms config so this is expected (the Czech (cs) one is `"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"`, at a glance Arabic appears to have the record with 6) **but** core translations are only set up for 2 - I somehow had it in my head that we'd adopted 3 - so maybe this should be held back for now.

Comment (garethrees) [reply lib/views/request/_bubble.html.erb]: Looks like we discussed this before and decided it wasn't really worth it for now https://github.com/mysociety/alaveteli/issues/3942.

Comment (garethrees) [reply lib/views/request/_bubble.html.erb]: Okay lets drop this change as I don't think we're 100% sure of the consequences and its too much effort to think about at the moment

**Comment by lizconlan** [comment]
ToDo: check whether the translations need to be regenerated after reverting 
bc14dd6

**Pull request merged**